### DISCUSSION TITLE
<meta name="darkreader-lock"> dont work

### DISCUSSION THREAD

**Discussion by marloeffler** [darkreader/darkreader#9853] [General]
Hey there, the `<meta name="darkreader-lock">` wont work on my site.

`<style class="darkreader darkreader--fallback" media="screen">html, body, body :not(iframe):not(div[style^="position:absolute;top:0;left:-"]) { background-color: #181a1b !important; border-color: #776e62 !important; color: #e8e6e3 !important; } html, body { opacity: 1 !important; transition: none !important; }</style>`

get still applied to my DOM

Browser:

Microsoft Edge
Version 105.0.1343.42 (Official Build) (64-Bit)

**Comment by Gusted** [comment]
Hi @marloeffler!

Please always open up a issue for bugs.

Which version of dark reader are you using?
